include/kframework/evm.md

```diff
@@ -10,11 +10,13 @@ This file only defines the local execution operations, the file `driver.md` will
 ```k
 requires "data.md"
 requires "network.md"
+requires "gas.md"
```

Comment:
Would `evm-types.md` be a more appropriate place to import this?

Reply:
I think `data.md` is actually probably the best place. Let's see if we can make this change in this PR.

Are we able to inline the `infinite-gas.md` into `gas.md`? Probably not I guess, because `infinite-gas.md` mentions things from `evm.md`.... I wonder if ti makes sense to move all the function declarations about gas calculations from `evm.md` to `gas.md` actually, that may be the most clean. I've made an issue about this: https://github.com/runtimeverification/evm-semantics/issues/1789. No need to address it in this PR.

Reply:
I tried refactoring `infinite-gas.md` in `gas.md`, but I couldn't avoid a cycle. You mean all the `Csstore` etc.? I think that makes sense